django/contrib/gis/db/models/__init__.py

```diff
@@ -11,4 +11,4 @@
 from django.contrib.gis.db.models.fields import (  # NOQA
     GeometryField, PointField, LineStringField, PolygonField,
     MultiPointField, MultiLineStringField, MultiPolygonField,
-    GeometryCollectionField)
+    GeometryCollectionField, RasterField)
```

Comment:
Not sure if this import should be conditional to the `supports_raster` db feature. Alternatively, the `RasterField.__init__` function could check for the db feature and give an error if instantiated with the wrong backend. Currently this would probably give some error from the DB that might not be explicit enough.